src/state/login.js

```diff
@@ -0,0 +1,8 @@
+import { atom } from 'recoil';
+
+const informacionUsuarioState = atom({
+  key: 'informacionUsuarioState',
```

Comment:
No veníamos poniéndole sufijo `state` a las keys, sí al nombre de la const:

```suggestion
  key: 'informacionUsuario',
```

Reply:
Al margen de esto, le pondría directamente `usuario` como nombre, me parece redundante lo de `informacion`.